src/screens/videos/mobile.js

```diff
@@ -26,8 +21,7 @@ export function VideosYouMightLikeMobile({
 
   return (
     <section
-      id="videos-screen"
-      data-name="Videos"
+      id="videos"
```

Comment:
should the VideosMobile and VideosDesktop have different `id`s? (not sure if they can be displayed at once or if it's a problem)

Reply:
I can answer this one (since I've been working on it): it used to be a yes, but it's no longer the case after my current PR (unless someone thinks it's a bad idea).

See: https://github.com/veganhacktivists/wildanimalsuffering/pull/25/files#diff-fceabc84abfe3f2cb540b44be13597140a5c946709af314c6c501de14dcc02d0

(Btw Tim I think you needed something like that for another screen iirc, so feel free to let me know what you think.)

Reply:
Good call. They should not have different ids, because we want to be able to permalink to them, and that should be agnostic to the width of the screen. Having multiple elements with the same id is not valid markup, so we should have only one `section#videos` element, and put the different responsive sections in there as a child.

Reply:
@Jeto143 the hook looks great!